packages/php-db-import-export/src/Backend/Snowflake/SnowflakeImportOptions.php

```diff
@@ -27,7 +33,8 @@ public function __construct(
         int $numberOfIgnoredLines = 0,
         bool $requireSameTables = self::SAME_TABLES_NOT_REQUIRED,
         bool $nullManipulation = self::NULL_MANIPULATION_ENABLED,
-        array $ignoreColumns = []
+        array $ignoreColumns = [],
+        array $autoCastTypes = [],
```

Comment:
to co sa má castovat sem dal jako option ať to jde nastavit z connection

Reply:
Smazal sem to, upravil sem test, vlatně je to potřeba po dalším testování jen pro 3 typy:         Snowflake::TYPE_VARIANT,
Snowflake::TYPE_OBJECT,
Snowflake::TYPE_ARRAY,
ostatní SNFLK nějak sám přecastuje z toho stringu.
Původně sem nad tím přemýšlal tak, že to půjde nastavit v connection a nebude potřeba šahat sem, ale vlastně je to blbost a chceš to mět tady odzkůšané.